BGAnimations/ScreenGameplay in/MeasureCounterAndModsLevel.lua

```diff
@@ -1,3 +1,67 @@
+-- Get the start/end of each stream and break sequence in our table of measures
+local GetStreamSequences = function(notesPerMeasure, notesThreshold)
+	local streamMeasures = {}
+	for i,n in ipairs(notesPerMeasure) do
+		if n >= notesThreshold then
+			table.insert(streamMeasures, i)
+		end
+	end
+
+	local streamSequences = {}
+	-- Count every measure as stream/non-stream.
+	-- We can then later choose how we want to display the information.
+	local measureSequenceThreshold = 1
```

Comment:
Why is this a variable? Would you ever use anything else but 1 for it?

Reply:
Talked offline. I agree we can use 1 everywhere, but I think this function can actually use a refactor to make it a bit more simpler if we're using that assumption. I added a TODO for now that I'll clean up later.